suffuse/inode_gen.go

```diff
@@ -0,0 +1,38 @@
+package suffuse
+
+type InodeGen struct {
+  fresh chan InodeNum
+}
+
+func NewRootInode(startInode uint64)*Inode {
+  ichan := make(chan InodeNum)
+  // Monotonically increments the counter without any
+  // effort to track or reuse.
+  go func() {
+    count := uint64(startInode)
```

Comment:
It seems `startInode` was typed `InodeNum`, if the type of argument was changed on purpose, you can remove the `uint64` call here.